README.md

```diff
@@ -220,7 +220,7 @@ class PageController extends ContentController
     {
         return [
             'Title' => 'Thank you for your submission',
-            'Content' => '<p>You have successfully submitted the form!</p>'
+            'Content' => DBHTMLText::create('<p>You have successfully submitted the form!</p>')
```

Comment:
Can you add the `use SilverStripe\ORM\FieldType\DBHTMLText` statement on line 205/206 as well please?

Reply:
Of course. Well, spotted.